### PR TITLE
Update caesar.txt

### PR DESCRIPTION
**Pull request by engineerscodes**

### CODE REVIEW

**Comment by engineerscodes** [comment]
updated

File: caesar.txt [hunk 1]
```diff
@@ -49,11 +59,11 @@ public class caesiro
 			 cipher=cipher+bal;
 		 }
 	 }
-	 
+	 System.out.println(cipher);
 	  return cipher;
   }
-  public static String decrypt(String text,int shift)
-  {
+  public  String decrypt(String text)
```
Comment (engineerscodes) [inline caesar.txt:57]: static is not needed

File: caesar.txt [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+ *  Tech/Project Lead Android Club
+ */
 import java.util.*;
 public class caesiro
 {
-  public static String encrypt(String text,int shift)
-  {
+	int shift=0;
+	
+  public  String encrypt(String text)
```
Comment (engineerscodes) [inline caesar.txt:8]: static is not needed

**Review by engineerscodes — COMMENTED** [review]
go through this

File: caesiro.java [hunk 3]
```diff
@@ -1,13 +1,5 @@
-/**
- * 
- */
-package AffineChiper;
```
Comment (engineerscodes) [inline caesiro.java:1]: **Changed file name and remove my Tag**

**Review by engineerscodes — COMMENTED** [review]
Go through this

File: caesar.txt (hunk 1 above)
Comment (Bala320) [reply caesar.txt:57]: Ok I'll check for it

**Pull request merged**